test/built-ins/Number/prototype/toFixed/return-values.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2016 The V8 Project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 20.1.3.3
+esid: sec-number.prototype.tofixed
+description: >
+  Return regular string values
+---*/
+
+// a bug in IE ? - Edge 14
+assert.sameValue((1.255).toFixed(2), "1.25");
```

Comment:
I don't have Edge right now to verify, but if the current assertion is enough to flag the bug, I would say both examples would be fine.